buildkite/bazelci.py

```diff
@@ -2861,11 +2861,16 @@ def bazelci_builds_nojdk_gs_url(platform, git_commit):
     return "gs://{}/artifacts/{}/{}/bazel_nojdk".format(bucket_name, platform, git_commit)
 
 
-def bazelci_builds_metadata_url():
+def bazelci_latest_build_metadata_url():
     bucket_name = "bazel-testing-builds" if THIS_IS_TESTING else "bazel-builds"
     return "gs://{}/metadata/latest.json".format(bucket_name)
 
 
+def bazelci_builds_metadata_url(git_commit):
+    bucket_name = "bazel-testing-builds" if THIS_IS_TESTING else "bazel-builds"
+    return "gs://{}/artifacts/metadata/{}/info.json".format(bucket_name, git_commit)
```

Comment:
why do we store these commits under the artifacts directory?

Reply:
I think @philwo recommended this in #1046 so the metadata will be stored under the same directory as the binaries.
binaries:    gs://{bucket_name}/artifacts/{platform}/{git_commit}
metadata: gs://{bucket_name}/artifacts/metadata/{git_commit}

Do you think it will be better if we have a directory next to artifacts for the metadata?

Reply:
I think @philwo recommended this in #1045 so the metadata will be stored under the same directory as the binaries.
binaries:    gs://{bucket_name}/artifacts/{platform}/{git_commit}
metadata: gs://{bucket_name}/artifacts/metadata/{git_commit}

Do you think it will be better if we have a directory next to artifacts for the metadata?

Reply:
Actually.. yup, maybe Florian is right. This URL template would be simpler and I can't think of any drawback:

`"gs://{}/metadata/{}.json".format(bucket_name, git_commit)`

What do you think?

Reply:
I think this is better too, I will update it.